draft-ietf-tls-tls13.md

```diff
@@ -2624,18 +2596,11 @@ for a server CertificateVerify would be:
 
 If sent by a server, the signature algorithm MUST be one offered in the
 client's "signature_algorithms" extension unless no valid certificate chain can be
-produced without unsupported algorithms (see {{signature-algorithms}}). Note that
-there is a possibility for inconsistencies here. For instance, the client might
-offer an ECDHE_ECDSA cipher suite but omit any ECDSA and EdDSA values from its
-"signature_algorithms" extension. In order to negotiate correctly, the server
-MUST check any candidate cipher suites against the "signature_algorithms"
-extension before selecting them. This is somewhat inelegant but is a compromise
-designed to minimize changes to the original cipher suite design.
-
-If sent by a client, the signature algorithm used in the
-signature MUST be one of those present in the
-supported_signature_algorithms field of the CertificateRequest
-message.
+produced without unsupported algorithms (see {{signature-algorithms}}).
+
+If sent by a client, the signature algorithm used in the signature
```

Comment:
I thought if the client didn't send a signature_algorithms, the server was supposed to abort.